cmd/validator/flags/flags.go

```diff
@@ -322,6 +322,26 @@ var (
 		Usage: "Enables more verbose logging for counting down to duty",
 		Value: false,
 	}
+
+	// ValidatorsProposerConfigDirFlag defines the path or URL to a file with proposer config.
+	ValidatorsProposerConfigDirFlag = &cli.StringFlag{
+		Name:  "validators-proposer-config-dir",
+		Usage: "remote local file path to load proposer configuration from",
+		Value: "",
+	}
+	// ValidatorsProposerConfigFlag defines the path or URL to a file with proposer config.
+	ValidatorsProposerConfigURLFlag = &cli.StringFlag{
+		Name:  "validators-proposer-config-url",
+		Usage: "remote URL file path to load proposer configuration from",
```

Comment:
In a vacuum, this doesnt tell me much it has to do with fee recipients. Maybe we can add more information to the usage or explain what this file should contain. Perhaps we can link to our docs portal once we have that ready